_posts/Summary Script, Ep. 1 Open Source Directions: Spyder

```diff
@@ -0,0 +1,94 @@
+This is the webinar that informs the community about what is going on with Open Source Python projects.  Today's project being highlighted is Spyder.
```

Comment:
This file will need similar changes as to the episode 0 one.

Reply:
This one is more so true to your first-person narration, but is this what you meant for me to fix in 0, or is it still wrong?

Reply:
I mean that you need to change the filename and add the front matter section here too